File: faculdade-marotinha-ngweb-master/src/app/aluno/models/aluno.model.ts
import { BaseResourceModel } from 'src/app/shared/models/base-resource.model';

export class AlunoModel extends BaseResourceModel {
	name?: string;
	age?: number;
	cpf?: string;
	phone?: number;
	turmas?: (string | number)[];

	constructor() {
		super();
	}
}

File: faculdade-marotinha-app-master/lib/alunos/models/alunoModel.dart
import 'package:faculdademarotinhaapp/commons/models/pessoaModel.dart';

class AlunoModel extends PessoaModel {

  AlunoModel({
    name,
    age,
    cpf,
    phone
  }) :
  super(
      name: name,
      age: age,
      cpf: cpf,
      phone: phone
  );

}